common/xbps-src/libexec/build.sh

```diff
@@ -26,6 +26,12 @@ for f in $XBPS_SHUTILSDIR/*.sh; do
     . $f
 done
 
+last="${XBPS_DEPENDS_CHAIN##*,}"
+case "$XBPS_DEPENDS_CHAIN" in
+    *,$last,*)
+        msg_error "Build-time cyclic dependency$last,${XBPS_DEPENDS_CHAIN##*,$last,} detected.\n"
```

Comment:
missing `;;` but maybe bash doesn't care at all with just a single case.

Reply:
`;;` is optional after last branch.